06-Customizing AzTS for your org/Extending AzTS/Prerequisites.md

```diff
@@ -3,7 +3,7 @@
 
 To enable the Control Metadata Editor tool (CMET), you need to configure a few application settings in for AzTS Function Apps/WebApps. This can be done either through the Azure portal or by using the helper script. Once these application settings are configured, please follow the steps mentioned [here](#access-to-cmet-control-metadata-editor-tool) to add users to Privileged editor roles.
 
-- Using Azure Portal
+### Option1: Using Azure Portal
```

Comment:
Two things - 1. Can you move the API, MA, WIP headers to a smaller font (by adding a '#'?) 
2. Can you fix MetaData to Metadata

![image](https://github.com/azsk/AzTS-docs/assets/88755870/68c289fa-813f-4f7a-b5fc-48f06166e172)

Reply:
Done